src/sign.rs

```diff
@@ -133,6 +138,12 @@ fn test_create_key() {
     )
     .unwrap();
     assert_eq!(key.y_i, Secp256k1Point::from_coor(&x, &y));
+
```

Comment:
I think we don't need test codes for ensuring the randomness in generating keys like this because generating 10 times at random does not guarantee that it will be safe to generate each time.

Reply:
BTW codes below need assertion like `assert_eq!` or `assert!' ?

Reply:
> generating 10 times at random does not guarantee that it will be safe to generate each time.
Yes. It was a bit useful for to notice elementary mistake. but I will remove it.

Reply:
Updated at 52fc964e5679a8f0d1ef60bbb4f72f3b19071154